src/App.jsx

```diff
@@ -34,12 +65,53 @@ function App() {
       <div className="card">
         <h1>Guess The Word 🚀</h1>
         <h3>Word Display</h3>
-        {generateWordDisplay()}
+        {isAlive && generateWordDisplay()}
         <h3>Guessed Letters</h3>
         {guessedLetters.length > 0 ? guessedLetters.toString() : "-"}
```

Comment:
```suggestion
        {guessedLetters.length ? guessedLetters.toString() : "-"}
```
above 0 is truthy, 0 is falsy